01_login.php

```diff
@@ -168,7 +168,7 @@
 
 							<br>
 							<div class="w3-center">
-								<button onclick="document.getElementById('id01').style.display='block'" class="w3-button w3-white w3-ripple w3-hover-grey">SIGN UP FOR SOUNDWAVE</button> 
+								<button onclick="document.getElementById('id01').style.display='block'" class="w3-button w3-orange w3-ripple w3-hover-grey">SIGN UP FOR SOUNDWAVE</button> 
```

Comment:
sign up button changed to orange to revisit the title colour; creates a connection